task/03-date-tasks.js

```diff
@@ -76,10 +77,15 @@ function isLeapYear(date) {
  *    Date(2000,1,1,10,0,0),  Date(2000,1,1,15,20,10,453)   => "05:20:10.453"
  */
 function timeSpanToString(startDate, endDate) {
-   throw new Error('Not implemented');
+   let hours, result; let deltaDate = new Date(endDate - startDate);
+   if(new Date(startDate).getUTCDay()!=new Date(endDate).getUTCDay()){
+      hours = deltaDate.getUTCDay()+24*(new Date(endDate).getUTCDay()-new Date(startDate).getUTCDay());
+      result = hours+":"+deltaDate.toISOString().slice(14,23);
+   }else{
+      result = deltaDate.toISOString().slice(11,23);
+   }
+   return result;
 }
```

Comment:
operator `===` or `!==` is always preferable 
also pls don't forget to use spaces